data/summary/store/store.go

```diff
@@ -260,6 +284,13 @@ func (r *Repo[T, A]) GetOutdatedUserIDs(ctx context.Context, page *page.Paginati
 		response.End = *userSummary.Dates.OutdatedSince
 	}
 
+	QueueLag.WithLabelValues(typ).Observe(time.Now().UTC().Sub(response.Start).Minutes())
```

Comment:
The duration ought to be measured in seconds and combined with the appropriate buckets configuration of the histogram.